elastic/kibana
# [Infra UI] Add 'host.name' field view for 'Logs' tab

closes #159560 

## 📝  Summary

This PR adds an inline view for the logs tab in the Hosts view, where the `host.name` column is statically added to the columns definition.

## ✅  Testing

1. Navigate to Stream page
2. Change the settings to have any Date Source and columns
3. Navigate to the logs tab in the Hosts view
4. Verify that the columns applied in step 2 aren't showing in the logs, and that `host.name` is showing.
5. Click the open in logs link and make sure that the settings show the host.name and notes an inline view is being used.


https://github.com/elastic/kibana/assets/11225826/a19f7969-31b3-40af-9e07-784631d6292f
